packages/legend-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -1248,7 +1248,6 @@ export const ClassFormEditor = observer(
       (tab: UML_EDITOR_TAB): (() => void) =>
       (): void => {
         editorState.setSelectedTab(tab);
-        editorState.setSelectedProperty(undefined);
```

Comment:
actually, I don't think we can remove this line here, since I suspect it would mean when we select a property and switch to another tab, say `Constraint`, the panel would still be there, which is weird. We need to hide it when switching between tabs of `class editor`